website/docs/components/alert/partials/content/content.md

```diff
@@ -0,0 +1,155 @@
+## General recommendations
+
+- Keep the title short, as this will be the most prominent element when users scan the alert.
+- Avoid ending the title with a period.
+- Alert descriptions should be short but clear enough to explain what’s happening. We recommend keeping messages under 90 characters.
+- For warning and critical alerts, guide the users on how to prevent or fix the issue.
+
+## Actions
+
+Use small buttons to avoid competing with other actions on the page. Use more than two actions sparingly.
+
+We recommend using the `secondary` button variant for primary actions and the `tertiary` button variant for secondary actions.
+
+<Hds::Alert @type="inline" as |A|>
+  <A.Title>Recommended button usage</A.Title>
+  <A.Description>Lorem ipsum dolar sit amet.</A.Description>
+  <A.Button @text="Your action" @color="secondary" @onClick={{this.noop}} />
+  <A.Button @color="tertiary" @icon="arrow-right" @iconPosition="trailing" @text="Tertiary" />
+</Hds::Alert>
+
+For content guidelines on actions, refer to [Button](/components/button) and 
+[Link](/components/link/standalone) documentation.
+
+### Usage of critical buttons
+
+Avoid using critical buttons in alerts. We handle the prominence and importance via the styling of the alert container itself. If you need to confirm that the user intended to interact with the action, consider displaying a confirmation modal.
+
+!!! Dont
+<Hds::Alert @type="inline" @color="critical" as |A|>
+  <A.Title>Critical button usage</A.Title>
+  <A.Description>Lorem ipsum dolar sit amet.</A.Description>
+  <A.Button @text="Your action" @color="critical" @onClick={{this.noop}} />
+</Hds::Alert>
+!!!
+
+### Links
+
+#### Within page and inline Alerts
+
+When linking to internal and external resources in the Alert, default to using a `secondary` [Standalone Link](/components/link/standalone) in the `actions` area. In the Ember component, these elements are passed as [contextual components](?tab=code#actions).
+
+<Hds::Alert @type="inline" @color="highlight" as |A|>
+  <A.Title>Links in alerts</A.Title>
+  <A.Description>Lorem ipsum dolar sit amet.</A.Description>
+  <A.Link::Standalone @icon="arrow-right" @iconPosition="trailing" @color="secondary" @text="Standalone link" @href="#" />
+</Hds::Alert>
+
+If linking within the Alert description, use the `secondary` [Inline Link](/components/link/inline) as the default. A common use case for this is when linking to multiple resources.
```

Comment:
```suggestion
Within the Alert description, use the `secondary` [Inline Link](/components/link/inline) as the default link type. A common use case for this is when linking to multiple resources.
```

Reply:
> as the default link type.

Though, for clarity I think this should be link **color** as that's the correct property name (even if it is also a type).

Reply:
Changed it